scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/PipelineSubscriber.kt

```diff
@@ -72,7 +74,7 @@ class PipelineSubscriber(
     suspend fun subscribe() {
         logger.info("will connect to ${upstreamHost}:${upstreamPort}")
         retry(grpcFailurePolicy + binaryExponentialBackoff(50..5_000L)) {
-            subscribePipelines()
+            subscribePipelines(kafkaConsumerGroupIdPrefix, namespace)
```

Comment:
:upside_down_face: No need to pass these parameters explicitly as they've `private val`s above; happy to change in a follow-up PR for expediency